CHANGELOG.md

```diff
@@ -1,14 +1,18 @@
+## HEAD (Unreleased)
+
+- Add root volume encryption validation to `encrypted-volumes` policy.
+
 ## 0.2.1 (2020-05-29)
 
-- Fix TypeScript compiler error when using the library (thanks @rsclarke-vgw!)
-  [#60](https://github.com/pulumi/pulumi-policy-aws/pull/60)
+- Fix TypeScript compiler error when using the library (thanks @rsclarke-vgw!)	
+  [#60](https://github.com/pulumi/pulumi-policy-aws/pull/60)	
 
-## 0.2.0 (2020-04-17)
+## 0.2.0 (2020-04-17)	
```

Comment:
```suggestion
## 0.2.0 (2020-04-17)
```

Reply:
Sorry, not sure how that got deleted in the first place 😬